auth/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 apply(from = rootProject.file("buildSrc/shared.gradle.kts"))
 
 plugins {
-    id("org.springframework.boot") version "3.3.5"
+    id("org.springframework.boot") version "3.3.4" // 3.3.5 has breaking bug for appending headers https://github.com/spring-projects/spring-framework/issues/33789
```

Comment:
Is this still true? If so, we should have a ticket on our board to look into it, but this ticket you linked is closed on their project.

Reply:
I can double check. That issue is closed because it's a part of the spring-security package which is just one of many in spring boot.